changelogs/fragments/remove-deprecated-datetime-methods.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - On Python 3 use datetime methods ``fromtimestamp`` and ``now`` with UTC timezone instead of ``utcfromtimestamp`` and ``utcnow``, which are deprecated in Python 3.12.
```

Comment:
It would be good to mention the new datetime compat features here as minor changes.